### PR TITLE
feat(add takes) - refactor TakesPackageService

### PR DESCRIPTION
**Pull request by amagdas**
- mark the package as "takes_package" only when needed

### CODE REVIEW

File: server/apps/archive/takes_package_service.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8; -*-
```
Comment (sivakuna-aap) [inline server/apps/archive/takes_package_service.py]: Since packages functionality is growing I think it's better to refactor to a different package.

Comment (amagdas) [inline server/apps/archive/takes_package_service.py]: You mean to move the archive_composite.py and takes_package_service.py to another folder module?

Comment (sivakuna-aap) [inline server/apps/archive/takes_package_service.py]: Yes, we can move other modules also if there are any related to package into one folder module.

Comment (amagdas) [inline server/apps/archive/takes_package_service.py]: Sounds good, will do.

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**